docs/framework/install/on-windows-10.md

```diff
@@ -11,15 +11,23 @@ ms.prod: .net-framework
 ---
 # Install the .NET Framework on Windows 10
 
-You may need the .NET Framework to run an app on Windows 10. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+The .NET Framework is required to run many applications on Windows. You can use the following instructions to install it. You may have arrived on this page after trying to run an application and seeing the following dialog on your machine.
+
+![This application could not be started](./media/this-application-could-not-be-started.png)
+
+These instructions will help you install the .NET Framework versions you need. The latest version is often your best choice. The latest version is the .NET Framework 4.7.1.
 
 ## .NET Framework 4.7.1
 
-The .NET Framework 4.7.1 is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10).
+* [Download the .NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral)
+
+The [.NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral) can be used to run applications built for [.NET Framework 4.0 or later](https://www.microsoft.com/net/download/framework).
 
-If you're on the Windows 10 Creators Update or Windows 10 Anniversary Update, you can install the .NET Framework 4.7.1 separately. If you're on an even earlier version of Windows 10, you need to upgrade to one of the supported Windows 10 versions to get the .NET Framework 4.7.1. For more information, see [Install the .NET Framework for developers](guide-for-developers.md).
+The .NET Framework 4.7.1 is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10) and [Windows Server 2016 Version 1709](https://docs.microsoft.com/windows-server/get-started/get-started-with-1709).
 
-The .NET Framework 4.7.1 supports apps built for .NET Framework 4.0 or later. Other versions of the .NET Framework are available from the [Download .NET Framework](https://www.microsoft.com/net/download/framework) webpage.
+If you're on the Windows 10 Creators Update (Version 1703), Windows 10 Anniversary Update (Version 1607), or Windows Server 2016, you can install the [.NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral) separately.
+
+If you're using Windows 10 1507 or 1511, then you need to upgrade to a later Windows 10 version, at least Windows 10 Anniverary Update or Windows Server 2016 or later to install the .NET Framework 4.7.1. The [.NET Framework 4.6.1](https://go.microsoft.com/fwlink/?LinkId=671729&source=ms-docs&desc=dotnet461) is the latest supported version on Windows 10 1507 and 1511.
```

Comment:
I thought of that. We have three choices:

- Make mention of both client and server.
- Split the server mentions into another doc.
- Make no mention of server.

I went with the first one. The last one seems like a bad idea. It stands to reason that non-developer users are relying on this same experience on servers. I need to check if I can distinguish Client vs. Server via the user agent string.

Thoughts?

Reply:
I agree, @richlander, that making no mention of server is a bad idea. My preference would be for a separate server doc. My basic concern is that it's not apparent to me that anyone wanting to install a version of the framework on server is going to look to a Windows 10 topic.

Reply:
The issue is that Windows Server 2016 and Windows 10 may have the same user agent string (still yet to prove) so the service I provide may send them to the same Win10 doc.

Reply:
In that case, I'd vote for changing the title -- Install the .NET Framework on Windows 10 or Windows Server 2016

Reply:
Actually, I hadn't noticed that that already is the TOC title. I think that it would be helpful if the topic title in each case corresponded to the TOC title, so that the topic is clearly about a client OS or a server OS.